### PR TITLE
Assignment1

### PR DESCRIPTION
**Pull request by Steffy2007**

### CODE REVIEW

**Comment by Steffy2007** [comment]
Close pull request

**Comment by Steffy2007** [comment]
Reopen pull request

**Comment by Steffy2007** [comment]
Added Assignment 1 and Assignment 3

File: Steffy/Assignment1.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+print(name)
+food = input("What is your favourite food?: ")
+print(name + " likes " + food)
+
+temp_celsius = input("Enter temperature in celsius: ")
+temp_f = (float(temp_celsius) * 9/5)+32
+print("Temperature in Fahrenheit = " + str(temp_f))
+
+#Fullname = input("Enter your full name: ")
+print(name.title())
```
Comment (meenanegi) [inline Steffy/Assignment1.py:37]: great job !

File: Steffy/Assignment3.1.py [hunk 2]
```diff
@@ -0,0 +1,9 @@
+firstname=input("Enter a First Name: ")
+print(firstname)
```
Comment (meenanegi) [inline Steffy/Assignment3.1.py:2]: you can do this by assigning a variable here
e.g
length_first_name = len(firstname)

File: Steffy/Assignment3.1.py [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+if len(firstname)<3:
```
Comment (meenanegi) [inline Steffy/Assignment3.1.py:3]: if length_first_name <3 
if. length_first_name > 20

**Review by meenanegi — APPROVED** [review]
Looks good !

**Pull request merged**